src/SB/Core/x/xCutscene.cpp

```diff
@@ -148,6 +148,105 @@ S32 xCutscene_LoadStart(xCutscene* csn)
     return 1;
 }
 
+S32 xCutscene_Update(xCutscene *csn, F32 dt)
+{
+    if ((csn->SndStarted == 0) && (csn->SndNumChannel != 0))
+    {
+        if (csn->SndNumChannel == 1)
+        {
+            xSndSetPitch(csn->SndHandle[0], 0.0f);
+        }
+        else if (csn->SndNumChannel == 2)
+        {
+            xSndStartStereo(csn->SndHandle[0], csn->SndHandle[1], 0.0f);
+        }
+        csn->SndStarted = 1;
```

Comment:
Opt for using the TRUE and FALSE macros for 0/1 when more semantically appropriate.

In this case, I would recommend doing `csn->SndStarted == FALSE` and `csn->SndStarted == TRUE`. Anything being used as a C-style boolean should use these when appropriate.

SndNumChannel and SndHandle aren't like that so just for cases where SndStarted is used would be enough.